README.md

```diff
@@ -34,7 +34,7 @@ http://groups.google.com/group/weed-file-system Seaweed File System Discussion G
 * Automatic compaction to reclaimed disk spaces after deletion or update
 * Servers in the same cluster can have different disk spaces, different file systems, different OS
 * Adding/Removing servers do not cause any data re-balancing
-* Optional  [filer server](https://code.google.com/p/weed-fs/wiki/DirectoriesAndFiles) provides "normal" directories and files via http
+* Optional  [filer server](https://github.com/chrislusf/seaweedfs/wiki/Directories-and-Files) provides "normal" directories and files via http
```

Comment:
seems the link not working. Is there any way to redirect? or just use
  https://github.com/chrislusf/weed-fs/wiki/Directories-and-Files